GameData/B9_Aerospace_HX/Patches/CRP/B9_Aerospace_HX-Engine_Instance.cfg

```diff
@@ -0,0 +1,357 @@
+				volume = 0.0 0.0
+				volume = 0.5 0.5
+				pitch = 0.0 0.1
+				pitch = 0.5 0.3
+				loop = true
+			}
+		}
+	}
+	
+	!MODULE[MultiModeEngine] {}
+	@MODULE[ModuleEnginesFX],0
+	{
+		@engineID = Karborundum
+		@runningEffectName = running_nswr
+		@powerEffectName = power_nswr
+		@heatProduction = 500
+		@maxThrust = 24000
+		!PROPELLANT,* {}
+		PROPELLANT:NEEDS[CommunityResourcePack]
+		{
+			name = Karborundum
+			ratio = 0.02
+			DrawGauge = True
+		}
+		PROPELLANT:NEEDS[CommunityResourcePack]
+		{
+			name = Water
+			ratio = 0.98
+			DrawGauge = True
+		}
+		PROPELLANT:NEEDS[CommunityResourcePack]
+		{
+			name = ElectricCharge
+            ratio = 15
+            DrawGauge = True
+		}
+		@atmosphereCurve
+		{
+			!key,* = null
+			key = 0 3000
+			key = 1 2820
+			key = 30 0.01
+		}
+	}
+	!MODULE[ModuleEnginesFX],1 {}
+	!MODULE[ModuleSurfaceFX],1 {}
+	
+	@#@TWEAKSCALEBEHAVIOR[Engine]/MODULE[TweakScale] { }
+	%MODULE[TweakScale]
+	{
+		type = stack
+		defaultScale = 7.5
+	}
+}
+
++PART[B9_Structure_HX_HPD1] // :NEEDS[CommunityResourcePack]
+{
+	@name = B9_Structure_HX_HPD3
+	@title = HX-HPD 3 Heavy Propulsion Device
+	@description = This variant of the HPD is a fusion rocket and primarily runs on Hydrogen. Compared to the other variants, this one trades a lot of its potential for thrust for immense Isp and still provides an afterburner or "wet" mode for that moment when efficiency is negligible.
+	
+	@EFFECTS
+	{
+		!running_lfo {}
+		!running_lava {}
+		!power_classic {}
+		!power_plasma {}
+		running_fusion
+		{
+			MODEL_MULTI_PARTICLE
+			{
+				name = bubbles
+				modelName = B9_Aerospace_HX/FX/KaRoarF2
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+			MODEL_MULTI_PARTICLE
+			{
+				name = core
+				modelName = B9_Aerospace_HX/FX/KaShockF2
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+			MODEL_MULTI_PARTICLE
+			{
+				name = stream
+				modelName = B9_Aerospace_HX/FX/KaStreamF2
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+		}
+		running_bluelava
+		{
+			MODEL_MULTI_PARTICLE
+			{
+				name = core
+				modelName = B9_Aerospace_HX/FX/blueCore
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+			MODEL_MULTI_PARTICLE
+			{
+				name = glimmer
+				modelName = B9_Aerospace_HX/FX/blueGlimmer
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+			MODEL_MULTI_PARTICLE
+			{
+				name = stream
+				modelName = B9_Aerospace_HX/FX/blueStream
+				transformName = nozzle_transform
+				emission = 0.0 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.2
+				speed = 1.0 1.0
+			}
+		}
+		// running_torch
```

Comment:
Any reason this is all commented out?

Reply:
It's a nice config, for one of the sets of plumes that I decided afterward that I wouldn't use. In case I have to add another propellant combo and another engine clone I can de-comment and move this.